### DISCUSSION TITLE
press matches exactly one non-modified key and no modified-keys.

### DISCUSSION THREAD

**Discussion by jgarte** [meow-edit/meow#443] [Q&A] [answered]
> ... press matches exactly one non-modified key and no modified-keys.

Hi,

I'm confused one what the above means in this blog post: https://esrh.me/posts/2021-12-18-switching-to-meow.html

Would someone be able to give an example of that concept?

**Comment by DogLooksGood** [comment, marked as answer]
In the key sequences in Keypad, the `SPC` can be used as literal prefix, so (In Keypad) `x SPC f` goes to `C-x f`. But if there's no `C-x C-f` defined, you can skip the `SPC`, just press `x f` for `C-x f`.

**Comment by jgarte** [comment]
Very cool! Thanks for explaining that further. It is much appreciated. I understand now 🎉 🤯 🤯 🎉

**Reply by idlip** [reply]
Heya @jgarte , there is no need to close this discussion as it wont appear by default on this page, you can mark it as solved tho.
So please open this, so others can see this on default (open discussion) page.

**Reply by jgarte** [reply]
Hi, re-opened. all best.
